history/src/main/scala/org/apache/spark/deploy/history/SimpleFsHistoryProvider.scala

```diff
@@ -141,7 +142,6 @@ private[history] class SimpleFsHistoryProvider(conf: SparkConf, clock: Clock)
     // actually read, we may never refresh the app.  FileStatus is guaranteed to be static
     // after it's created, so we get a file size that is no bigger than what is actually read.
     Utils.tryWithResource(EventLoggingListener.openEventLog(logPath, fs)) { in =>
-      logInfo("Trying to replay bus: $isCompleted, $eventsFilter")
```

Comment:
Why remove?

Reply:
I added this as a debug log and forgot to remove it.

Reply:
so couldn't we just log it to... debug? :)

Reply:
If we do that, fix up the grammar on it, but it looks more like it was used to just print some variables while Peter was working on it.

Reply:
The content really isn't important on it. If I recall, I didn't even us the log line in testing.